_data/history.yaml

```diff
@@ -0,0 +1,56 @@
+# Chronological history of the Nottingham New Theatre
+
+- year: 1927
+  title: Formation of Dramsoc
+  description: "The University College Nottingham Dramatic Society (Dramsoc) performed its first play in 1928: A.A. Milne's *Mr Pim Passes By*. <br /> The society offered members an opportunity to read, discuss, produce and enact plays."
```

Comment:
I agree with you. I think they must get stripped as the line breaks don't get processed in markdownify process.